Packs/NationalVulnerabilityDatabaseFeed/Integrations/NationalVulnerabilityDatabaseFeed/NationalVulnerabilityDatabaseFeed.py

```diff
@@ -74,9 +75,24 @@ def parse_cve_data(item=None) -> dict:
                             }
                         )
                     fields['cvss3'] = cvss_v3_data
+
+                    cvss_data = []
+                    for k, v in cvss_v3.items():
+                        cvss_data.append(
+                            {
+                                "metrics": camel_case_to_underscore(k).replace("_", " ").title(),
+                                "value": v
+                            }
+                        )
+                    fields['cvsstable'] = cvss_data
+                    # additional fields to write to CVE default layout
                     fields['cvssvector'] = cvss_v3.get('vectorString')
+                    fields['cvss'] = cvss_v3.get('baseScore')
+                    fields['cvssscore'] = cvss_v3.get('baseScore')
```

Comment:
Is the `cvss` & `cvssscore` the same value?

Reply:
yes they are the same score.   The default CVE layout has both listed as fields and there is a field driven automation that leverage the cvss score to show a colored number at the top of the layout.  The current integrations were not writing to CVSS Score so the layout was always blank and not showing value.  So I added the integration to write to both which helps fill out the layout better OOTB.